cmd/dep/status.go

```diff
@@ -424,7 +432,7 @@ type MissingStatus struct {
 	MissingPackages []string
 }
 
-func runStatusAll(ctx *dep.Ctx, out outputter, p *dep.Project, sm gps.SourceManager) (hasMissingPkgs bool, errCount int, err error) {
+func runStatusAll(ctx *dep.Ctx, out outputter, p *dep.Project, sm gps.SourceManager, cmd *statusCommand) (hasMissingPkgs bool, errCount int, err error) {
```

Comment:
Let's not do this and go back to a separate function `runOld`. This is the default status run function. Maybe we would rename it to "default" in the future. You can refer `ensure.go` to see how we have different functions for different run modes.